auth/src/main/java/org/aerogear/auth/credentials/OIDCCredentials.java

```diff
@@ -67,8 +145,21 @@ public boolean isAuthorized() {
      * Returns stringified JSON for the OIDCCredential.
      * @return Stringified JSON OIDCCredential
      */
-    public String serialise() {
-        return this.authState.jsonSerializeString();
+    public String serialize() throws JSONException {
+        JSONObject jsonCredential = new JSONObject()
```

Comment:
Every time a variable is not supposed to be assigned again, it should be declared as final.